docs/spec_draft.md

```diff
@@ -1247,6 +1248,39 @@ defined and one of the following:
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.is_finite
+
+### Semantics
+
+Performs element-wise check whether the value in `x` is finite (i.e. is neither
++inf, -inf, nor NaN) and produces a `y` tensor.
```

Comment:
NaN can also be negative.
Should we cover this in the example and remove it from here (and write the operand in a pretty way with inf, nan etc), or we can let it be here for now but write the operand in a pretty format in the example.
Let me know your thoughts

Reply:
> NaN can also be negative.

Some PRs back, it was decided that we do not include explicit examples of negative NaN because our spec nor IEEE-754 differentiates the two. Take a look at #142

> and write the operand in a pretty way with inf, nan etc

There is an issue with this addressed [here](https://github.com/openxla/stablehlo/issues/289#issuecomment-1309665935).